.github/workflows/test.yml

```diff
@@ -42,6 +42,9 @@ jobs:
       - uses: conda-incubator/setup-miniconda@v2
         with:
           miniconda-version: "latest"
+          python-version: ${{ matrix.python-version }}
+          channel-priority: strict
+          activate-environment: test-environment
```

Comment:
To be clear, I'm not sure if we want this or not. IIUC this should make the conda environment used by the default shell the same as the one created by `doit env_create` later on. Right now there are two separate conda environment (one named `test-environment` and one named `test`) and I'm not sure if that is intentional or not